dist_tables/working_with_distributed_tables.rst

```diff
@@ -20,14 +20,3 @@ Typically, you should choose that column as the distribution column which is the
   Citus also allows joining on non-distribution key columns by dynamically repartitioning the tables for the query. Still, joins on non-distribution keys require shuffling data across the cluster and therefore aren’t as efficient as joins on distribution keys.
 
 The best option for the distribution column varies depending on the use case and the queries. In general, we find two common patterns: (1) **distributing by time** (timestamp, review date, order date), and (2) **distributing by identifier** (user id, order id, application id). Typically, data arrives in a time-ordered series. So, if your use case works well with batch loading, it is easiest to distribute your largest tables by time, and load it into Citus in intervals of N minutes. In some cases, it might be worth distributing your incoming data by another key (e.g. user id, application id) and Citus will route your rows to the correct shards when they arrive. This can be beneficial when most of your queries involve a filter or joins on user id or order id.
```

Comment:
Can we amend this section as well, and remove the content around distribution by 'time'? I think we can have a short blurb here about 'distributing by tenant or application' and the other being 'distributing by user-id', and very briefly discuss pros/cons here.